compiler/rustc_errors/src/emitter.rs

```diff
@@ -1266,22 +1266,37 @@ impl EmitterWriter {
             }
             self.msg_to_buffer(&mut buffer, msg, max_line_num_len, "note", None);
         } else {
+            let mut label_width = 0;
             // The failure note level itself does not provide any useful diagnostic information
             if *level != Level::FailureNote {
                 buffer.append(0, level.to_str(), Style::Level(*level));
+                label_width += level.to_str().len();
             }
             // only render error codes, not lint codes
             if let Some(DiagnosticId::Error(ref code)) = *code {
                 buffer.append(0, "[", Style::Level(*level));
                 buffer.append(0, &code, Style::Level(*level));
                 buffer.append(0, "]", Style::Level(*level));
+                label_width += 2 + code.len();
```

Comment:
Can we say something like `": ".len()` presuming I've guessed correctly as to the meaning of this 2? (Or just a comment on at least one of these).

Reply:
This `2` belongs to the two `[` `]` surrounding the `code`. There's another `2` below for the `": "`. Kept these magic numbers very close to their causes to try and make them obvious, but I guess they aren't?